### ISSUE TITLE
Crash after placing Crafting terminal:

### ISSUE THREAD

**Issue by Iomega0318** [AppliedEnergistics/Applied-Energistics-2#1474]
I had a crafting terminal on a wall not connected to anything and have been waiting for channels to get fixed so I could disable them, after disabling channels and logging in I picked up the crafting terminal and placed it down, I crashed but it didn't give an accurate crash report however on trying to log back in I now get the following crash and cannot log back into the server:

http://paste.ubuntu.com/11196205/

**Comment by yueh** [comment]
Cannot reproduce it.

**Comment by Iomega0318** [comment]
Why do I seem to find all the bugs you can't reproduce, I don't really know what to do then since I cannot log in. I will try rebooting the server when I get home.

**Comment by Iomega0318** [comment]
Why do I seem to find all the bugs you can't reproduce, I don't really know what to do then since I cannot log in. I will try rebooting the server when I get home. I assume it is related to AE though since that's what the error log reported.

**Comment by Iomega0318** [comment]
I thought this might be related to forge microparts or waila so I updated both as well as AE to Stable v3 and still getting the crash, here is another log: http://paste.ubuntu.com/11235075/

Is there anything else I can do to try and find out why I am crashing?

**Comment by Iomega0318** [comment]
Found what this was actually related to! This crash was occurring because of my previously mentioned issue with being able to craft the "Unnamed" cables!! Because they used the same id and I had one in my inventory when I placed the terminal the inventory sorter tried to fill my hand with the "same" item which was the bugged cable therefore trying to render it.. However because it's not supposed to exist it was crashing me, I had to remove the item from my inventory using an NBT Editor to get back in. So this can now be closed thank you. Also I will try to experiment more with the unnamed cable and get you some more information.
